designs/discount-pricing.md

```diff
@@ -0,0 +1,55 @@
+# Discounted Pricing Support
+
+## Overview
+
+Karpenter is currently unaware of any discounted pricing, such as volume discounts or reserved instances/savings plans, which can lead to more expensive instances being chosen. For example an on demand instance with a savings plan discount may cost less than a spot instance of the same type. This pricing is apparently only available from the "payer" account, not any other child account API's for pricing.
```

Comment:
```suggestion
Karpenter is currently unaware of any discounted pricing, such as volume discounts or reserved instances/savings plans, which can lead to more expensive instances being chosen. For example an on demand instance with a savings plan discount may cost less than a spot instance of the same type. On AWS, this pricing is apparently only available from the "payer" account, not any other child account API's for pricing.
```

:thought_balloon: could Karpenter learn to assume an IAM role into the AWS account where discounts are visible?